other/edge-server/backend/data_sources/mongodb-atlas/todo/Item/rules.json

```diff
@@ -6,7 +6,7 @@
             "name": "readAndWriteAll",
             "apply_when": {
                 "%%user.type": {
```

Comment:
Engineering pointed out in one of my docs PRs that we can replace this expression with something simpler:

`"%%user.type": "edge"`